laske_export/tests/conftest.py

```diff
@@ -15,6 +16,64 @@ def pytest_configure():
     }
 
 
+def pytest_runtest_setup(item):
+    if item.get_closest_marker("ftp") and os.environ.get(
+        "TEST_FTP_ACTIVE", False
+    ) not in [1, "1", True, "True"]:
```

Comment:
Environment variables are stored as strings. Value cannot be `1` or `True`.
```suggestion
    ) not in ["1", "True"]:
```